api/osrd_infra/serializers.py

```diff
@@ -170,26 +170,17 @@ class Meta:
 
 class TrainScheduleSerializer(ModelSerializer):
     class Meta:
-        model = TrainScheduleModel
-        exclude = [
-            "mrsp",
-            "base_simulation",
-            "eco_simulation",
-            "modes_and_profiles",
-        ]
+        model = TrainSchedule
+        fields = "__all__"
 
 
 class StandaloneSimulationSerializer(Serializer):
     class Schedule(ModelSerializer):
         class Meta:
-            model = TrainScheduleModel
+            model = TrainSchedule
             exclude = [
                 "timetable",
                 "path",
-                "mrsp",
-                "base_simulation",
-                "eco_simulation",
-                "modes_and_profiles",
             ]
```

Comment:
I think this is supposed to exclude the outputs from the serialization when creating the payload to the back-end. We probably need to add something in there. But I'm not 100% sure and can't really check.